company-meghanada.el

```diff
@@ -63,8 +63,8 @@
   (company-mode t)
   (set (make-local-variable 'company-backends) nil)
   (set (make-local-variable 'company-transformers) nil)
-  (set (make-local-variable 'company-idle-delay) 0)
-  (set (make-local-variable 'company-minimum-prefix-length) company-meghanada-prefix-length)
+  (when company-meghanada-prefix-length
+    (set (make-local-variable 'company-minimum-prefix-length) company-meghanada-prefix-length))
   (setq company-meghanada-trigger-regex (format company-meghanada--trigger company-meghanada-prefix-length company-meghanada-prefix-length))
```

Comment:
If `company-meghanada-prefix-length` value is "nil" It seems that `company-meghanada-trigger-regex` is broken. 
Could you please check?